github/src/main/kotlin/gropius/sync/github/TimelineGrabber.kt

```diff
@@ -0,0 +1,100 @@
+package gropius.sync.github
+
+import com.apollographql.apollo3.ApolloClient
+import gropius.sync.github.generated.TimelineReadQuery
+import gropius.sync.github.generated.TimelineReadQuery.Data.Companion.metaData
+import gropius.sync.github.generated.TimelineReadQuery.Data.Node.Companion.asIssue
+import gropius.sync.github.generated.fragment.TimelineItemData
+import gropius.sync.github.generated.fragment.TimelineItemData.Companion.asNode
+import gropius.sync.github.model.IssueInfo
+import gropius.sync.github.model.TimelineItemDataCache
+import gropius.sync.github.repository.IssueInfoRepository
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import kotlinx.coroutines.reactive.asFlow
+import kotlinx.coroutines.reactor.awaitSingle
+import org.bson.types.ObjectId
+import org.springframework.data.mongodb.core.*
+import org.springframework.data.mongodb.core.query.Criteria
+import org.springframework.data.mongodb.core.query.Query
+import org.springframework.data.mongodb.core.query.Update
+import java.time.OffsetDateTime
+
+/**
+ * Implementation of Grabber to retrieve timeline items and cache them in the database
+ */
+class TimelineGrabber(
```

Comment:
missing documentation for params, inner class

Reply:
cafe999d546dae9937ec2e811c2d873877c50a22

Reply:
the parameter if the inner class is still not documented